src/frameworks/angular/index.ts

```diff
@@ -36,7 +36,7 @@ export const docsUrl = "https://firebase.google.com/docs/hosting/frameworks/angu
 
 const DEFAULT_BUILD_SCRIPT = ["ng build"];
 
-export const supportedRange = "14 - 17";
+export const supportedRange = "15 - 18";
```

Comment:
btw @jamesdaniels shouldn't it be `16-18` based on https://angular.dev/reference/releases#actively-supported-versions?